src/app/shared/shared-settings-forms/units-form/units-form.component.html

```diff
@@ -157,7 +157,7 @@
             <div class="col-4">
                 <div class="input-group">
                     <input class="form-control" type="number" step="any" min="0" formControlName="waterUse"
-                        name="waterUse" (input)="saveChanges()" (focus)="focusField()">
+                        name="waterUse" (change)="saveChanges()" (focus)="focusField()">
```

Comment:
if the input type="number" we want to use the (input) event 
if the input is select or checkbox then use change